src/store/index.js

```diff
@@ -85,26 +100,47 @@ export default new Vuex.Store({
         alert(e);
       }
     },
-    async nextMoviesPage({commit}, {page, query}) {
+    async getNextMoviesPage({ commit }, { page, query }) {
       const options = {
-        params: {api_key: process.env.VUE_APP_API_KEY, query, language: "ru", page}
-      }
+        params: {
+          api_key: process.env.VUE_APP_API_KEY,
+          query,
+          language: "ru",
+          page,
+        },
+      };
       if (query) {
         try {
-          const response = await axios.get("https://api.themoviedb.org/3/search/movie", options)
-          commit("NEXT_MOVIES_PAGE", response.data)
+          const response = await axios.get(
+            "https://api.themoviedb.org/3/search/movie",
+            options
+          );
+          commit("NEXT_MOVIES_PAGE", response.data);
         } catch (e) {
           alert(e);
         }
       } else {
         try {
-          const response = await axios.get("https://api.themoviedb.org/3/movie/popular", options)
-          commit("NEXT_MOVIES_PAGE", response.data)
+          const response = await axios.get(
+            "https://api.themoviedb.org/3/movie/popular",
+            options
+          );
+          commit("NEXT_MOVIES_PAGE", response.data);
         } catch (e) {
           alert(e);
         }
       }
-    }
+    },
+    updateSearch({ commit }, val) {
+      commit("UPDATE_SEARCH", val.trim());
```

Comment:
You can move the `.trim()` inside the `SET_SEARCH_QUERY` (`UPDATE_SEARCH`) mutation, since you'll be removing the `updateSearch` action as I described above.